Medicina/CDB/alma/DataBlock/DMed/DMed.xml

```diff
@@ -7,61 +7,64 @@
 				IP="192.168.51.28"
 				PORT="5003">
 
-    <Configuration name="sole">
+    <Configuration name="default">
     	<Board number="6">
-    		<Attenuator ID="0" Value="10.0"/>
+    		<Attenuator ID="0" Value="0.0"/>
     		<Attenuator ID="1" Value="7.0"/>
     		<Attenuator ID="2" Value="0.0"/>
     		<Attenuator ID="3" Value="10.0"/>    		
     	</Board>
     	<Board number="8">
-    		<Attenuator ID="0" Value="26.0"/>
-    		<Attenuator ID="1" Value="10.0"/>
-    		<Attenuator ID="2" Value="29.0"/>
+    		<Attenuator ID="0" Value="0.0"/>
+    		<Attenuator ID="1" Value="8.0"/>
+    		<Attenuator ID="2" Value="0.0"/>
     		<Attenuator ID="3" Value="10.0"/> 
     	</Board>
     	<Board number="9">
-    		<Attenuator ID="0" Value="15.0"/>
+    		<Attenuator ID="0" Value="13.0"/>
     		<Attenuator ID="1" Value="10.0"/>
-    		<Attenuator ID="2" Value="13.0"/>
+    		<Attenuator ID="2" Value="16.0"/>
     		<Attenuator ID="3" Value="10.0"/>     	
     	</Board>
      	<Board number="10">
-    		<Attenuator ID="0" Value="23.0"/>
+    		<Attenuator ID="0" Value="16.0"/>
     		<Attenuator ID="1" Value="10.0"/>
-    		<Attenuator ID="2" Value="22.0"/>
+    		<Attenuator ID="2" Value="14.0"/>
     		<Attenuator ID="3" Value="10.0"/>      	
     	</Board>  	
     </Configuration>
 
 
-    <Configuration name="KKC">
+    <Configuration name="sun">
     	<Board number="6">
-    		<Attenuator ID="0" Value="10.0"/>
-    		<Attenuator ID="1" Value="10.0"/>
-    		<Attenuator ID="2" Value="10.0"/>
+    		<Attenuator ID="0" Value="0.0"/>
+    		<Attenuator ID="1" Value="7.0"/>
+    		<Attenuator ID="2" Value="0.0"/>
     		<Attenuator ID="3" Value="10.0"/>    		
     	</Board>
     	<Board number="8">
-    		<Attenuator ID="0" Value="10.0"/>
-    		<Attenuator ID="1" Value="10.0"/>
-    		<Attenuator ID="2" Value="10.0"/>
+    	   <Attenuator ID="0" Value="0.0"/>
+    		<Attenuator ID="1" Value="8.0"/>
+    		<Attenuator ID="2" Value="0.0"/>
     		<Attenuator ID="3" Value="10.0"/> 
     	</Board>
-    	<Board number="9">
-    		<Attenuator ID="0" Value="10.0"/>
+   	<Board number="9">
+    		<Attenuator ID="0" Value="28.0"/>
     		<Attenuator ID="1" Value="10.0"/>
-    		<Attenuator ID="2" Value="10.0"/>
+    		<Attenuator ID="2" Value="26.0"/>
     		<Attenuator ID="3" Value="10.0"/>     	
     	</Board>
      	<Board number="10">
-    		<Attenuator ID="0" Value="10.0"/>
+    		<Attenuator ID="0" Value="11.0"/>
+    		<Attenuator ID="1" Value="10.0"/>
+    		<Attenuator ID="2" Value="22.0"/>
+    		<Attenuator ID="3" Value="10.0"/>      	
+    	</Board> 	
+    </Configuration>ID="0" Value="10.0"/>
```

Comment:
I guess this line will not be parsed correctly

Reply:
File corrected accordingly